python/pyspark/ml/feature.py

```diff
@@ -2610,6 +2610,9 @@ def setParams(self, inputCol=None, outputCol=None, stopWords=None, caseSensitive
         Sets params for this StopWordRemover.
         """
         kwargs = self._input_kwargs
+        if locale is None:
+            sc = SparkContext._active_spark_context
+            kwargs['locale'] = sc._gateway.jvm.org.spark.ml.util.LocaleUtils.getDefaultLocale()
```

Comment:
We can keep this default local, and use it many times instead of call to JVM form Python every time.

Reply:
@viirya You mean... `locale=SparkContext._active_spark_context.(...)`over `locale=None` with ugly if statement, right?

Reply:
I was meant that if `sc._gateway.jvm.org.spark.ml.util.LocaleUtils.getDefaultLocale()` returns the same every time, shall we cache it and reuse it? But after re-thinking it, maybe we won't call this method with `locale is None` frequently. So it maybe fine.